zipline/data/minute_bars.py

```diff
@@ -269,12 +270,14 @@ def __init__(
         start_session,
         end_session,
         minutes_per_day,
+        version,
     ):
         self.calendar = calendar
         self.start_session = start_session
         self.end_session = end_session
         self.ohlc_ratio = ohlc_ratio
         self.minutes_per_day = minutes_per_day
+        self.version = version
```

Comment:
Can you remind me the lifecycle here?  We instantiate a `BcolzMinuteBarMetadata`, which will have version from what was last written to disk.  When we write this instance to disk, it will have that previous version?